.github/workflows/release.yaml

```diff
@@ -148,7 +148,7 @@ jobs:
         with:
           status: ${{ job.status }}
           fields: repo,workflow,action,eventName
-          text: "A new Grype release is ready to be manually published: https://github.com/anchore/grype/releases"
+          text: "A new Grype release has been published: https://github.com/anchore/grype/releases"
```

Comment:
It would be nice to include the direct reference, like in Syft: https://github.com/anchore/syft/blob/main/.github/workflows/release.yaml#L151

```
text: "A new Grype release has been published: https://github.com/anchore/grype/releases/tag/${{ github.ref_name }}"
```